packages/@aws-cdk/integ-tests-alpha/README.md

```diff
@@ -555,6 +557,7 @@ const describe = testCase.assertions.awsApiCall('StepFunctions', 'describeExecut
   totalTimeout: Duration.minutes(5),
   interval: Duration.seconds(15),
   backoffRate: 3,
+  logRetention: logs.RetentionDays.ONE_WEEK,
```

Comment:
could we add this property to an existing integration test?

Reply:
Thank you for instructive comment! 
I think, it might take time, but I will try it!